tools/backport_pr.py

```diff
@@ -2,20 +2,27 @@
 """
 Backport pull requests to a particular branch.
 
-Usage: backport_pr.py branch [PR] [PR2]
+Usage: backport_pr.py [org/repository] branch [PR] [PR2]
```

Comment:
Because it's easy to missused and backport a PR we don't want to backport, and why would it stay in IPython/IPython ?  Plus the `backport_pr.py org/repo/pr` seem much more natural to me as a syntax if you've [set up your git correctly](https://gist.github.com/piscisaureus/3342247)